### PR TITLE
add missing doctests to chapter_seq_annot.tex

### PR DESCRIPTION
**Pull request by mdehoon**
This pull request adds the missing doctests to chapter_seq_annot.tex.

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

File: Doc/Tutorial/chapter_seq_annot.tex [hunk 1]
```diff
@@ -132,21 +132,19 @@ \subsection{SeqRecord objects from FASTA files}
 used to loop over all the records in a file as \verb|SeqRecord| objects. The \verb|Bio.SeqIO| module
 has a sister function for use on files which contain just one record which we'll use here (see Chapter~\ref{chapter:seqio} for details):
 
-%TODO - line wrapping for doctest?
```
Comment (peterjc) [inline Doc/Tutorial/chapter_seq_annot.tex:135]: How bad does this look in the PDF output? I forget the timing, maybe switching to minted for the code blocks helps?

**Comment by peterjc** [comment]
Inserting line breaks to make the PDF (and fixed width HTML) pretty breaks the doctests.

**Comment by peterjc** [comment]
I like the idea of using minted options to improve the layout (with minor tweak to the test framework).

**Comment by peterjc** [comment]
I recommend running ``python test_Tutorial.py`` locally, it is more verbose than via ``run_tests.py``

**Comment by mdehoon** [comment]
@peterjc 
> I recommend running `python test_Tutorial.py` locally, it is more verbose than via `run_tests.py`
The bug shows up on Windows only. On mac all tests pass.

**Comment by mdehoon** [comment]
Gave up

**Pull request closed without merging**